content/manuals/docker-hub/download-rate-limit.md

```diff
@@ -140,6 +141,10 @@ charged at an on-demand rate. You can [scale your
 limit](../subscription/scale.md) or [upgrade](../subscription/change.md) to get
 a higher limit.
 
+We calculate storage based on each unique image layer (or “digest”) stored within a repository. Only unique layers contribute to your storage usage, which means if multiple images in a single repository share the same layer, you’re only billed once for that layer. However, if the same layer is used across different repositories, it will count separately in each one.
```

Comment:
```suggestion
Docker Hub calculates storage based on each unique image layer (or “digest”) stored within a repository. Only unique layers contribute to your storage usage, which means if multiple images in a single repository share the same layer, you’re only billed once for that layer. However, if the same layer is used across different repositories, it will count separately in each one.
```